packages/react-native/React/Base/RCTBundleURLProvider.h

```diff
@@ -142,6 +146,6 @@ RCT_EXTERN void RCTBundleURLProviderAllowPackagerServerAccess(BOOL allowed);
 + (NSURL *)resourceURLForResourcePath:(NSString *)path
                          packagerHost:(NSString *)packagerHost
                                scheme:(NSString *)scheme
-                                query:(NSString *)query;
+                           queryItems:(NSArray<NSURLQueryItem *> *)queryItems;
```

Comment:
Same reasoning as above. This is breaking. :( 

Can we duplicate the API, deprecate the `query:(NSString *)` and have it call the other one? 

Or is this change required at all? I feel that is orthogonal from the `inlineSourceMap` change, probably it can go into a different PR.

Reply:
It's orthogonal, and I can separate it out. But it will always be breaking, I think. So is it worth me making a new PR?

Reply:
I guess I can make an additive change here too.